transaction.go

```diff
@@ -104,8 +105,23 @@ func (tx *mysqlTx) register() (int64, error) {
 	var branchID int64
 	var err error
 	for retryCount := 0; retryCount < config.GetATConfig().LockRetryTimes; retryCount++ {
+		itemsBuffer := tx.mc.ctx.sqlUndoItemsBuffer
+		SKIP_CHECK_LOCK := true
```

Comment:
Please use camel format for variables
```suggestion
		skipCheckLock := true
```

Reply:
好的，已经改了

Reply:
这个 pr 等 seata-golang 那边代码提交了，我一起合

Reply:
这个 pr 等 seata-golang 那边提交了，我一起合

Reply:
> 这个 pr 等 seata-golang 那边提交了，我一起合

好的，已经提了